README.md

```diff
@@ -1116,7 +1116,7 @@ Fortio is written in the [Go](https://golang.org) language and includes a scalab
 The [fhttp/](fhttp/) package includes a very high performance specialized HTTP 1.1 client.
 You may find fortio's [logger](log/logger.go) useful as well.
 
-You can run the histogram code standalone as a command line in [histogram/](histogram/), a basic echo HTTP server in [echosrv/](echosrv/), or both the HTTP echo and gRPC ping server through `fortio server`, the fortio command line interface lives in this top level directory [fortio_main.go](fortio_main.go)
+You can run the histogram code standalone as a command line in [histogram/](histogram/), a basic echo HTTP server in [echosrv/](echosrv/), or both the HTTP echo and gRPC ping server through `fortio server`, the fortio command line interface lives in this top level directory [fortio_main.go](cli/fortio_main.go)
```

Comment:
Thanks for noticing. let’s use main.go though as that’s the entry point and also the important part is location of the “main main” at the root

Reply:
I made that change.